Scripts/databasebackup.ps1

```diff
@@ -0,0 +1,23 @@
+#Set-AWSCredentials -AccessKey AKIA3JDVQNWWHFSBKU55 -SecretKey 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 -StoreAs myAWScredentials
+Set-AWSCredentials myAWScredentials
+
+$server = 'dbregressiontest.database.windows.net'
+$backupPath = 'C:\Program Files (x86)\AZURESQL'
+$s3bucket = 'lansa-us-east-1/db-regression-test/backups/150050/'
+$region = 'us-east-1'
+
+$databases = Invoke-Sqlcmd -ServerInstance $server -Username "lansa" -Password "Pcxuser@122robg" -Query "SELECT [name]
+FROM master.dbo.sysdatabases where [name]='test'"
```

Comment:
The LansaVersion tag should be used to find the database to backup. Only ONE database should be found. Its an error if its 0 or greater than 1.
The LansaVersion should be a parameter to the script. In this case it should be 'Test' as thats the only one you have thats a valid DB regression Test database.
Then that parameter can be used EVERYWHERE the LansaVersion is required - e.g. the S3 bucket and locating the database.

Reply:
Hi Rob, When I check the database`s after creating a new one and also existing SQL server there is a default master database exist. so, we will have a min of 1 database.